pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -993,6 +995,10 @@ func buildDefaultCluster(env *model.Environment, name string, discoveryType apiv
 		Type: discoveryType,
 	}
 
+	// Default to 1 request per connection. This effectively disables HTTP keep-alives for both H1 and H2
+	// and eliminates a bunch of 503 issues
+	cluster.MaxRequestsPerConnection = &types.UInt32Value{Value: 1}
```

Comment:
Same - did you do any benchmark with the impact ? Numbers ?

Reply:
I don't think we can ignore this.  We'd need performance and efficiency numbers before making this sort of change